omnibus/package-scripts/updater-rpm/posttrans

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+readonly INSTALL_DIR=/opt/datadog/updater
```

Comment:
seems `OCI_PRODUCT` is `datadog-updater`, could it be an issue here?

Reply:
the OCI package path and rpm/deb are currently different ones, /opt/datadog/updater is the right one for rpm